lib/ui/onboarding_page.dart

```diff
@@ -0,0 +1,147 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:paintroid/ui/color_schemes.dart';
+import 'package:paintroid/ui/onboarding_screens/screen1.dart';
+import 'package:paintroid/ui/onboarding_screens/screen3.dart';
+import 'package:paintroid/ui/onboarding_screens/screen4.dart';
+import 'package:paintroid/ui/onboarding_screens/screen5.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:smooth_page_indicator/smooth_page_indicator.dart';
+
+import 'onboarding_screens/screen2.dart';
+
+class OnboardingPage extends StatefulWidget {
+  final Widget? navigateTo;
```

Comment:
I think it would be cleaner if you hand over a VoidCallback here from outside of the class to call in here.
Let me know if I should explain further

Reply:
ah.. you mean a function for the navigation part in the `finish()` method, right?

Reply:
You declare the function wherever the class is called and hand it over as a boidcallback, then you just call the voidcallback :)